index.d.ts

```diff
@@ -23,7 +23,7 @@ interface ConfigParams {
      * REQUIRED. The secret(s) used to derive an encryption key for the user identity in a session cookie.
```

Comment:
Guessing this should now read `OPTIONAL`

Reply:
No, it's still required but you can set the value in your env and not in the config object.

Reply:
Hmm, feels weird to have an optional property on this type but then say it's required in the docs. Wouldn't that confuse people?

Reply:
The application needs a value, that part is a requirement, but you can set it in the config object (which the TS defs affect) or you can set it in an env variable and skip the config key. Either way it is used by the app.

Reply:
https://github.com/auth0/express-openid-connect#getting-started